pkg/controllers/disruption/consolidation_test.go

```diff
@@ -76,6 +78,12 @@ var _ = Describe("Consolidation", func() {
 					v1beta1.CapacityTypeLabelKey: mostExpensiveOffering.Requirements.Get(v1beta1.CapacityTypeLabelKey).Any(),
 					v1.LabelTopologyZone:         mostExpensiveOffering.Requirements.Get(v1.LabelTopologyZone).Any(),
 				},
+				OwnerReferences: []metav1.OwnerReference{{
+					APIVersion: v1beta1.Group + "/v1beta1",
+					Kind:       "NodePool",
+					Name:       nodePool.Name,
+					UID:        uuid.NewUUID(),
```

Comment:
I added it because the feature uses the OwnerRef to locate the NodePool for the Node/NodeClaim. It isn't necessary for these tests to pass since TGP isn't considered specifically for consolidation at the moment.

Reply:
Removing all of these additions since I've modified the code to find the NodePool using the label on the NodeClaim instead.